src/main/java/org/generationcp/middleware/pojos/file/FileMetadata.java

```diff
@@ -0,0 +1,159 @@
+package org.generationcp.middleware.pojos.file;
+
+import org.generationcp.middleware.pojos.AbstractEntity;
+import org.generationcp.middleware.pojos.dms.ExperimentModel;
+
+import javax.persistence.*;
```

Comment:
with our settings, I think we should not be importing the whole package, right? @nahuel-soldevilla 
Implementing equals, hashCode and toString is missing in this and another classes. 
Does it worth to at least implement it in the entity?

Reply:
>with our settings, I think we should not be importing the whole package, right?
Right. I created the entity from the table using intellij plugin (POJO generator), and it generates it that way.
I cleaned up the imports and added toString.
I'm not sure if it's worth to add hashCode and equals, perhaps we can wait for a scenario where we actually needed

Reply:
:+1: